From 72be5a8accbb0d2f6dd9e25f6b996a91ddb25812 Mon Sep 17 00:00:00 2001
From: Dorian Eydoux <deydoux@student.42lyon.fr>
Date: Wed, 3 Dec 2025 10:45:23 +0000
Subject: [PATCH] Set EXTRAVERSION to "-thor_kernel" in Makefile
To: linux-kbuild@vger.kernel.org

Signed-off-by: Dorian Eydoux <deydoux@student.42lyon.fr>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6f0e72ff4d0c..79c3bf2c0e79 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 18
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION = -thor_kernel
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
-- 
2.50.1

